src/briefcase/integrations/subprocess.py

```diff
@@ -493,7 +493,7 @@ def _stream_output_thread(self, popen_process):
                 output_line = ensure_str(popen_process.stdout.readline())
                 if output_line:
                     self.tools.logger.info(output_line)
-                elif output_line == "":
+                else:
```

Comment:
The old version of this left an open coverage case for `readline()` returning None. This shouldn't ever happen; if it does, it's probably safe to assume that the process has ended, so I've made this a bucket else case.